client/src/elm/App/View.elm

```diff
@@ -1085,6 +1086,15 @@ viewUserPage page deviceName site features geoInfo reverseGeoInfo model configur
                             |> Html.map (MsgLoggedIn << MsgPageMessagingCenter nurseId)
                             |> flexPageWrapper configured.config model
 
+                    MessagingGuide ->
+                        let
+                            ( nurseId, nurse ) =
+                                loggedInModel.nurse
+                        in
+                        Pages.MessagingGuide.View.view model.language
+                            |> Html.map (MsgLoggedIn << MsgPageMessagingCenter nurseId)
```

Comment:
Only Msg page needs is `SetActivePage`, which can be used at top level (App.Model).
No need to make use of MessagingCenter page for that. 

See example of Clinical page:

![Selection_722](https://github.com/user-attachments/assets/043d7281-d939-4530-864a-d7baa36f0ea8)
![Selection_721](https://github.com/user-attachments/assets/1c6fe1a8-4d81-4069-a1e6-28a9bb152955)